src/components/Button.tsx

```diff
@@ -189,6 +189,13 @@ const variantStyles: (contrast: boolean) => Record<
     disabledStyles: Css.lightBlue300.if(contrast).lightBlue700.$,
     focusStyles: Css.bshFocus.if(contrast).boxShadow(`0 0 0 2px ${Palette.White}`).$,
   },
+  secondaryText: {
+    baseStyles: Css.gray900.add("fontSize", "inherit").$,
+    hoverStyles: Css.bgGray100.$,
+    pressedStyles: Css.gray900.$,
+    disabledStyles: Css.bgWhite.gray400.$,
+    focusStyles: Css.gray900.$,
+  },
```

Comment:
**question:** Right now, each of our Buttons has a `contrast` variation. We probably don't need this immediately, but would be good to add a `TODO: Handle contrast variant`.

Reply:
got it. I did ask Andrea about this, but agreed should probably put a note about coming back to this later